build.rs

```diff
@@ -9,9 +9,4 @@ fn main() {
             .unwrap_or_default(),
         None => println!("no need for pb"),
     }
-
-    Command::new("cargo")
```

Comment:
`use std::process::Command;` should be deleted too.

Reply:
Indeed 😅 
Done